data/xml/2024.gitt.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2024.gitt">
+  <volume id="1" ingest-date="2024-09-16" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of the 2nd International Workshop on Gender-Inclusive Translation Technologies</booktitle>
+      <editor><first>Beatrice</first><last>Savoldi</last></editor>
+      <editor><first>Janiça</first><last>Hackenbuchner</last></editor>
+      <editor><first>Luisa</first><last>Bentivogli</last></editor>
+      <editor><first>Joke</first><last>Daems</last></editor>
+      <editor><first>Eva</first><last>Vanmassenhove</last></editor>
+      <editor><first>Jasmijn</first><last>Bastings</last></editor>
+      <publisher>European Association for Machine Translation (EAMT)</publisher>
+      <address>Sheffield, United Kingdom</address>
+      <month>June</month>
+      <year>2024</year>
+      <url hash="3b54253c">2024.gitt-1</url>
+      <venue>gitt</venue>
```

Comment:
The gitt.yaml file has "First workshop" in the venue title; should be changed to just "workshop".

<img width="1068" alt="image" src="https://github.com/user-attachments/assets/044563f4-2634-4cce-a028-a748c7c544d3">

Reply:
Addressed